Makefile

```
@@ -1,15 +1,20 @@
+SCHEMAS_VERSION=`cat .schemas-version`
 DESIGN_SYSTEM_VERSION=`cat .design-system-version`
 
 clean:
 	rm -rf schemas
 	rm -rf templates/components
 	rm -rf templates/layout
 
+load-schemas:
+	./scripts/load_release.sh onsdigital/eq-questionnaire-schemas $(SCHEMAS_VERSION)
+
 load-design-system-templates:
 	./scripts/load_release.sh onsdigital/design-system $(DESIGN_SYSTEM_VERSION)
 	./scripts/load_print_styles_from_cdn.sh $(DESIGN_SYSTEM_VERSION)
 
 build: load-design-system-templates
+	make load-schemas
 	make translate
```

Minor but since they are all make commands, this could just be:
```suggestion
build: load-design-system-templates load-schemas translate
```